TabulateSmarterTestAdminPackage/Processors/Loader/TestSpecificationErrorProcessor.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.Xml.XPath;
+using TabulateSmarterTestAdminPackage.Common.Enums;
+using TabulateSmarterTestAdminPackage.Common.Validators;
+using TabulateSmarterTestAdminPackage.Processors.Specification;
+using TabulateSmarterTestAdminPackage.Utility;
+
+namespace TabulateSmarterTestAdminPackage.Processors.Loader
+{
+    internal class TestSpecificationErrorProcessor : TestSpecificationProcessor
+    {
+        internal TestSpecificationErrorProcessor(XPathNavigator navigator, PackageType expectedPackageType) : base(navigator, expectedPackageType)
+        {}
+
+        internal new bool IsValidPublisher()
+        {
+            var validators = new ValidatorCollection(new List<Validator>
+            {
+                new RequiredStringValidator(ErrorSeverity.Severe, null),
+                new MaxLengthValidator(ErrorSeverity.Severe, 100)
+            });
```

Comment:
If you switch to the `ValidatorCollection` that inherits from `List<Validator>` then you can instantiate these like:

```
new ValidatorCollection() {
                new RequiredStringValidator(ErrorSeverity.Severe, null),
                new MaxLengthValidator(ErrorSeverity.Severe, 100)
            };
```